README.md

```diff
@@ -10,6 +10,8 @@ This module simply initializes socket.io and configures it in a way that single
 npm install webrtc-scalable-broadcast
 
 node .\node_modules\webrtc-scalable-broadcast\server.js
```

Comment:
Actually, Windows accepts forward slashes too.
You should always use forward slashes for consistency.